source/picolibrary/testing/unit/indicator.cc

```diff
@@ -0,0 +1,23 @@
+/**
+ * picolibrary
+ *
+ * Copyright 2020-2021, Andrew Countryman <apcountryman@gmail.com> and the picolibrary
+ * contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/**
+ * \file
+ * \brief picolibrary::Testing::Unit::Indicator implementation.
+ */
+
+#include "picolibrary/testing/unit/indicator.h"
```

Comment:
Please add this file to the `picolibrary` source file list in `source/CMakeLists.txt` so that it will be built. It should only be added to the source file list if unit testing is enabled.

https://github.com/apcountryman/picolibrary/blob/025b2923def008041ff05701d8112806795dfbfe/source/CMakeLists.txt#L78-L91